library/aws/policies_v2.ts

```diff
@@ -0,0 +1,326 @@
+import * as AR from "../../providers/aws/resources.ts";
+import { ArnSecret } from "./secrets.ts";
+
+export interface NamedPolicy {
+  name: string;
+  policy: {
+    Statement: Statement[],
+    [key: string]: unknown;
+  };
+}
+
+export interface Statement {
+  Action: unknown;
+  Effect: unknown;
+  [key: string]: unknown;
+}
+
+export function newNamedPolicy(
+  name: string,
+  statements: Statement[],
+): NamedPolicy {
+  return {
+    name,
+    policy: {
+      Version: "2012-10-17",
+      Statement: statements,
+    },
+  };
+}
+
+export function combineNamedPolicies(
+  name: string,
+  namedPolicies: NamedPolicy[],
+): NamedPolicy {
+  const Statement: Statement[] = [];
```

Comment:
`const statement`

Reply:
It's `Statement` to have the correct case when used in the JSON construction.